napa_app/napa_app/sdk/AlchemyAPI.cs

```diff
@@ -961,7 +961,12 @@ private string DoRequest(HttpWebRequest wreq, AlchemyAPI_BaseParams.OutputMode o
 
                 if (string.IsNullOrEmpty(xml))
                     throw new XmlException ("The API request returned back an empty response. Please verify that the url is correct.");
-			
+
+                if (outputMode == AlchemyAPI_BaseParams.OutputMode.JSON)
```

Comment:
So the weird thing is it automatically assumes you wanted XML. If you look at the rest of that file, it checks to see if its XML or RDF, but only after it's already tried to parse the string that came back. If the string comes back as JSON, I don't want it to try to do the

XmlDocument xmlDoc = new XmlDocument ();
xmlDoc.LoadXml(xml);

lines because it'll break.That's why I put it before those lines. All that said, there is probably a cleaner way of organizing the code so that the check isn't just floating there.